src/ui.text-alert/composables/attrs.composable.js

```diff
@@ -30,12 +42,17 @@ export function useAttrs(props) {
     ),
   );
 
-  const bodyClasses = computed(() => cvaBody({ size: props.size }));
+  const bodyClasses = computed(() => setColor(cvaBody({ size: props.size }),props.color));
+
+  const buttonClasses = computed(() => setColor(cvaButton,props.color));
```

Comment:
same, why you need this for buttons and icons, if you already have the prop color there?

Reply:
need to discuss